src/Type/Scalar/Upload.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace WPGraphQL\Type\Scalar;
+
+use GraphQL\Error\UserError;
+use GraphQL\Error\InvariantViolation;
+use GraphQL\Utils\Utils;
+use UnexpectedValueException;
+
+/**
+ * Class Upload
+ *
+ * Registers Upload scalar type to WPGraphQL schema.
+ */
+class Upload {
+
+	/**
+	 * Keys found for a file in the $_FILES array to validate against.
+	 *
+	 * @var string[]
+	 */
+	public static $validationFileKeys = [ 'name', 'type', 'size' ];
+
+	/**
+	 * Register the scalar Upload type.
+	 *
+	 * @return void
+	 */
+	public static function register_type() {
+		register_graphql_scalar(
+			'Upload',
+			[
+				'description'  => __( 'The `Upload` special type represents a file to be uploaded in the same HTTP request as specified by [graphql-multipart-request-spec](https: //github.com/jaydenseric/graphql-multipart-request-spec).', 'wp-graphql' ),
+				'serialize'    => fn() => static::serialize(),
```

Comment:
arrow functions are only supported in PHP 7.4 or later.  These should be changed to long-form

Reply:
Good catch!